hoomd/hpmc/IntegratorHPMCMono.h

```diff
@@ -1695,7 +1701,7 @@ void IntegratorHPMCMono<Shape>::growAABBList(unsigned int N)
     the tree needs to be rebuilt or if the current tree can be used.
 
     buildAABBTree() relies on the member variable m_aabb_tree_invalid to work correctly. Any time particles
-    are moved (and not updated with m_aabb_tree->update()) or the particle list changes order, m_aabb_tree_invalid
+    are moved (and not updated with buildAABBTree->update()) or the particle list changes order, m_aabb_tree_invalid
```

Comment:
```suggestion
    are moved (and not updated with m_aabb_tree->update()) or the particle list changes order, m_aabb_tree_invalid
```
The comment was correct as it was.